development/extensions/tutorial_php_events_and_extending_extensions.rst

```diff
@@ -231,6 +238,19 @@ contained in the array keys):
       'ext/acme/demo/images/image2.png' => 'demo',
   );
 
+.. note::
+  The method ``find_from_extension`` used above will only search in that specific
+  extension. If you want to search for images in all extensions, you use ``suffix``,
+  ``directory`` and ``find`` instead:
```

Comment:
That's a bit more complicated:

 - find() will search in both core and extensions depending on what is configured (if a path is configured for the core it will search in core and extensions, otherwise only extensions)
 - extension_*() methods will configure the options for the extensions only
 - core_*() methods will configure the options for the core only
 - suffix/prefix/...() methods will configure the options for both (options, not path)
 - core_path() will configure the path for the core